dev-libs/actor-framework/actor-framework-0.15.3.ebuild

```diff
@@ -0,0 +1,88 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit flag-o-matic cmake-multilib
+
+bench_ref="d824310d2c0354c9d41fd0458eab88629dc29a94"
+opencl_ref="cc4e4ef5ad92e3ee78e2f2897027e5f48cbc8e93"
+python_ref="a754a79f193463fcbb712bafac098f8e2d85931e"
+
+DESCRIPTION="The C++ Actor Framework (CAF)"
+HOMEPAGE="https://actor-framework.org/"
+SRC_URI="https://github.com/actor-framework/actor-framework/archive/${PV}.tar.gz -> ${P}.tar.gz
+	benchmarks? ( https://github.com/actor-framework/opencl/archive/${opencl_ref}.tar.gz -> CAF_${PV}-benchmarks.tar.gz )
+	opencl? ( https://github.com/actor-framework/opencl/archive/${opencl_ref}.tar.gz -> CAF_${PV}-opencl.tar.gz )
+	python? ( https://github.com/actor-framework/python/archive/${python_ref}.tar.gz -> CAF_${PV}-python.tar.gz )"
+LICENSE="|| ( Boost-1.0 BSD )"
+SLOT="0/15.3"
+KEYWORDS="~amd64 ~x86"
+IUSE="benchmarks boost debug doc examples +mem_management opencl python static test tools"
+
+DEPEND="${RDEPEND}
+	doc? ( app-doc/doxygen )"
+RDEPEND="boost? ( dev-libs/boost[${MULTILIB_USEDEP}] )
+	net-misc/curl[${MULTILIB_USEDEP}]
+	opencl? ( virtual/opencl[${MULTILIB_USEDEP}] )"
+
+src_unpack() {
+	unpack ${A}
+	for i in opencl python; do
```

Comment:
No.
With opencl and python use, the sources get copied into the source tree, as ```libcaf_${i}```, while for benchmarks, the directory ```benchmark-*somehash*``` gets renamed to ```benchmark```.

Explanation: python and opencl stuff gets compiled and installed. Benchmark stuff comprises source files in a variety of languages and gets installed (e.g. copied) uncompiled.